types/apple-signin-api/index.d.ts

```diff
@@ -31,17 +35,18 @@ declare namespace AppleSignInAPI {
     }
     // https://developer.apple.com/documentation/signinwithapplejs/authi
     interface AuthI {
-        init: (config: ClientConfigI) => Promise<void>;
-        signIn: (signInConfig?: ClientConfigI) => Promise<SignInResponseI | SignInErrorI>;
+        init: (config: ClientConfigI) => void;
+        signIn: (signInConfig?: ClientConfigI) => Promise<SignInResponseI>;
```

Comment:
The error type of a promise can't be specified, and it will never resolve with the error, so having the type specified doesn't do anything.